package.json

```diff
@@ -7,6 +7,7 @@
   },
   "scripts": {
     "lint": "eslint .",
+    "pretest": "node test/helpers/pretest.js",
```

Comment:
I see, can you do the setup as part of the cucumber feature set via setup/teardown hooks: https://github.com/cucumber/cucumber-js/blob/master/docs/support_files/hooks.md#hooks